README.md

```diff
@@ -614,15 +601,13 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   end
   ~~~
 
-* Avoid the use of `attr`. Use `attr_reader` and `attr_accessor` instead.
+* Prefer `attr_reader` and `attr_accessor` over `attr`.
 
-* Avoid the usage of class (`@@`) variables due to their "nasty" behavior in
-  inheritance.
+* Avoid class (`@@`) variables.
```

Comment:
here I also like the little reference to the reason, esp. people for new to ruby it might be nice so to see it's not just arbitrary rules but there's actual danger in using these.

Reply:
Same as above. Consistency is the reason why you should follow this rule.